.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Description
+
+<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
+
+Fixes # (issue)
```

Comment:
We typically use JIRA for internal bug fixes, but I think we should try to use Github Issues for open-source projects.